gridformat/gridformat.hpp

```diff
@@ -8,6 +8,8 @@
 #ifndef GRIDFORMAT_GRIDFORMAT_HPP_
 #define GRIDFORMAT_GRIDFORMAT_HPP_
 
+#include <type_traits>
+
```

Comment:
```suggestion
```

Reply:
actually, we had used `std::true_type` already, so the additional include is fine...